js/components/urlBar.js

```diff
@@ -159,6 +159,18 @@ class UrlBar extends ImmutableComponent {
     })
     // escape key handling
     ipc.on(messages.SHORTCUT_ACTIVE_FRAME_STOP, this.onActiveFrameStop.bind(this))
+    ipc.on(messages.CERT_ERROR, (e, details) => {
+      // Sometimes the cert error fires before the active frame location
+      // has been updated, so wait 100ms.
+      window.setTimeout(() => {
```

Comment:
that doesn't seem bad, just post an issue pls though to handle this in electron later.